README.html

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
```

Comment:
Not sure what this file is :) I think it should be removed.

Reply:
True, probably leftovers from some build process :)